### PR TITLE
lowers sandbag health

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
sandbags have too much health and can be repaired and setup up pretty quickly

## Why It's Good For The Game
sandbags too good

## Changelog
:cl:
balance: sandbag intregrety halved
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -199. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by SplinterGP** [comment]
i'd reduce them to 300 instead of 200, since chaz nerfed their price per sandbag

**Comment by EvilMelonTheSecond** [comment]
Sandbags aren't too good metal cades are too bad. I would prefer being able to repair the cades but I simply cannot take the risk of using metal cades because those are paper thin and fall apart when more than a single xeno focuses them(Bonus points if a shrike keeps flinging me while trying to repair). Also remember that marines have no high grade cade option(Unless you count plasteel for 80 fucking req points, no thanks) so the cade lines cannot prepare for late game sieges where the marines might be scattered and cadelines aren't defended as much, marines losing bases instantly to random xeno wolfpacks is what will keep happening. 

I don't understand the "sandbags too strong" idea everybody has for some reason, it's literally the only option if you want anything more than a paper line that holds for 5 seconds.

**Comment by Pariah919** [comment]
no point to bothering

File: code/game/objects/structures/barricade.dm [hunk 1]
```diff
@@ -1065,7 +1065,7 @@
 	name = "sandbag barricade"
 	desc = "A bunch of bags filled with sand, stacked into a small wall. Surprisingly sturdy, albeit labour intensive to set up. Trusted to do the job since 1914."
 	icon_state = "sandbag_0"
-	max_integrity = 400
+	max_integrity = 200
```
Comment (Pariah919) [inline code/game/objects/structures/barricade.dm:1068]:
```suggestion
	max_integrity = 300
```

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-323). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**